README.md

```diff
@@ -22,12 +22,16 @@ Or install it yourself as:
 
 Within a Spotlight application with `spotlight-dor-resources` installed, you could index a set of records the ["harvestdor"](https://github.com/sul-dlss/harvestdor) or ["harvestdor-indexer"](https://github.com/sul-dlss/harvestdor-indexer) way, or from the rails console using:
 
+For access to a Rails console with the gem loaded up for testing purposes , you can use the engine-cart generated app:
+
+```
```

Comment:
When giving directions for the console, convention is to prefix the lines with e.g. `$ `. I also think ```console may trigger appropriate syntax highlighting here?